web/ts/TUMLiveVjs.ts

```diff
@@ -485,6 +485,14 @@ export function attachCurrentTimeEvent(videoSection: VideoSections) {
     });
 }
 
+export function currentTimeToHMS() {
```

Comment:
I don't really understand why we are not taking just a timestamp and safe that ?

Reply:
When I first implemented video sections I started with timestamps but then I transitioned into the 3-tuple of `(h,m,s)` because I ran into all sort of problems. For one, it's easier to sort numbers. 😅